docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mambaorg/micromamba:1.5.9
+FROM mambaorg/micromamba:1.5.10
```

Comment:
As of yet there's no micromamba 2.0 image, but I imagine it'll come out shortly.